urlextract/urlextract_core.py

```diff
@@ -698,7 +671,7 @@ def gen_urls(self, text, check_dns=False, get_indices=False):
         Creates generator over found URLs in given text.
 
         :param str text: text where we want to find URLs
-        :param bool check_dns: filter results to valid domains
+        :param bool check_dns: DEPRECATED filter results to valid domains
```

Comment:
I am not sure if we want to deprecate it. I still think we should keep this slow DNS check here. 
I believe there might be users that could use that. Because when we remove it, we will basically remove DNS checks for users which are using  `gen_urls()` and I think it might not be good. 

What do you think?

Reply:
As this is already a public method I think we could just keep it as you said. No problem

Reply:
Thank you! Could you remove the DEPRECATED text, please.